ui/app/components/auth-jwt.js

```diff
@@ -165,9 +162,7 @@ export default Component.extend({
     } catch (e) {
       return this.handleOIDCError(e);
     }
-    let token = resp.auth.client_token;
-    this.onToken(token);
-    yield this.onSubmit();
+    yield this.onSubmit(null, null, resp.auth.client_token);
```

Comment:
maybe a comment here about why null null, what those params do?

Reply:
Well I forgot to add a comment before merging! I had it in my editor and didn't push the commit :face_with_spiral_eyes:. I'll keep it in my mind and slide it in a future PR.